### ISSUE TITLE
Install in linux

### ISSUE THREAD

**Issue by ghost** [shargon/Xploit#4]
Hello guy, how setup xploit in linux ? It work on linux ?

**Comment by shargon** [comment]
Have you installed mono-complete?

**Comment by ghost** [comment]
Yes, can you do an make files with monodevolop. See my project made in C# for your code sources -> https://github.com/teeknofil/Crisis-Wordlist-Generator

PS : I from french, excuse me bad english.
